wuzz.go

```diff
@@ -1800,3 +1880,33 @@ func main() {
 		log.Panicln(err)
 	}
 }
+
+func exportJSON(r Request) []byte {
+	requestMap := map[string]string{
+		URL_VIEW:             r.Url,
+		REQUEST_METHOD_VIEW:  r.Method,
+		URL_PARAMS_VIEW:      r.GetParams,
+		REQUEST_DATA_VIEW:    r.Data,
+		REQUEST_HEADERS_VIEW: r.Headers,
+	}
+
+	request, err := json.Marshal(requestMap)
+	if err != nil {
+		return []byte{}
+	}
+	return request
+}
+
+func exportCurl(r Request) []byte {
+	var headers, params string
+	for _, header := range strings.Split(r.Headers, "\n") {
+		if header == "" {
+			continue
+		}
+		headers = fmt.Sprintf("%s -H '%s'", headers, header)
```

Comment:
Proper shell escaping should be used here. After a quick search, this lib can be the right solution: https://github.com/alessio/shellescape .

Reply:
shellescape seems to be a bit simplistic for our usecase ( https://github.com/alessio/shellescape/blob/master/shellescape.go#L30 )
A test give this example : 
```
curl http://google.com?yo=2&pwet=you -d '{"mama":"coucou"}' -H 'Accept: yes'
```
To be turned into
```
'curl http://google.com?yo=2&pwet=you -d '"'"'{"mama":"coucou"}'"'"' -H '"'"'Accept: yes'"'"''
```
What kind of problem would you try to avoid?

Reply:
> What kind of problem would you try to avoid?

Code execution. E.g.`'; rm -rf ./; echo '` as header value

> `'curl http://google.com?yo=2&pwet=you -d '"'"'{"mama":"coucou"}'"'"' -H '"'"'Accept: yes'"'"''`

this is strange, I tried it in 6d45930 and it produces the following output:

`curl  -H 'Accept: yes' -H 'Content-Type: application/x-www-form-urlencoded' -X POST -d '{"mama":"coucou"}' 'http://google.com/?yo=2&pwet=you'`

If you agree that this is a good solution, I'll merge your commit with 6d45930 from the local `exportAsCurl` branch

Reply:
You’re right, I completely misused the shellescape lib, I tried to quote the whole command instead
https://github.com/asciimoo/wuzz/commit/6d459300a0868bf39f74535a8a1909292e21f782 Looks good to me !